linux_os/guide/system/kernel_build_config/kernel_config_default_mmap_min_addr/rule.yml

```diff
@@ -22,6 +22,7 @@ identifiers:
     cce@rhel8: CCE-88160-7
     cce@rhel9: CCE-88161-5
 
+platform: x86_64_arch
```

Comment:
Do we need to think about `s390x` or `ppc64le`?

Reply:
I think we don't need to, I think that would go above the scope of ANSSI.

In ANSSI 2.0, there are 4 requirements in the section 5.3.2 Hardware dependent configuration.
- R24 for 32 bit x86
- R25 for x86_64
- R26 for ARM
- R27 for ARM 64 bit

There are no specific requirements for other architectures.

Now all 4 requirements in that section check for CONFIG_DEFAULT_MMAP_MIN_ADDR. But for the two 32 bit architectures we have decided that we won't support them and we have add a note about that to the controls file.

So we could think and invent some rules for s390x or ppc64le but then we would have a problem to which ANSSI requirement we would map these rules.

Reply:
But now I'm not sure whether we should add an architecture platform also to other rules included in these requirements.